api/src/main/java/com/kodemon/api/dto/UserAuthDTO.java

```diff
@@ -6,6 +6,7 @@
  * @author Matej Poklemba
  */
 public class UserAuthDTO {
```

Comment:
It was there in example project probably to identify user, but we have userName which is unique so we probably wont need it here.